v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/modbus_evse_client.py

```diff
@@ -421,6 +421,9 @@ async def set_inactive(self):
         """To be called when charge_mode in UI is (switched to) Stop
         Do not cancel polling, the information is still relevant.
         """
+        if self.client is None:
```

Comment:
Incidental finding while debugging:
This is invoked when the modbus connection may not be created yet.

Reply:
What should be done?